packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -437,6 +437,34 @@ class ToolbarOptions {
 ///  * When the virtual keyboard pops up.
 /// {@endtemplate}
 ///
+/// {@template flutter.widgets.editableText.accessibility}
+/// ## Troubleshooting Common Accessibility Issues
+///
+/// ### Customizing User Input Accessibility Announcements
+///
+/// To customize user input accessibility announcements triggered by text
+/// changes, use [SemanticsService.announce] to make the desired
+/// accessibility announcement.
+///
+/// {@tool snippet}
+/// On iOS, the on-screen keyboard may announce the most recent input
+/// incorrectly when we insert a thousands separator to a currency value text
+/// field. The following example demonstrates how to suppress the default
+/// accessibility announcements by always announcing the content of the text
+/// field as a US currency value.
+///
+/// ```dart
+/// onChanged: (String newText) {
```

Comment:
Maybe put an entire TextField or editable text?

Reply:
I'm setting this up as a template to be used in `TextField` `CupertinoTextField` as well so I'm trying to avoid using the name of the exact widget. I removed the tool marks maybe that will prevent the analyzer from analyzing this.